### PR TITLE
Log a user in after registration if the user needs to complete 2FA setup

### PR DESCRIPTION
**Pull request by cobusc**

### CODE REVIEW

File: authentication_service/views.py [hunk 1]
```diff
@@ -137,6 +137,14 @@ def form_valid(self, form):
         # Let the user model save.
         response = super(RegistrationView, self).form_valid(form)
 
+        # When we need to show the option to enable 2FA the newly created
+        # user must be logged in.
+        if self.security == "high" or self.request.GET.get("show2fa") == "true":
+            new_user = authenticate(username=form.cleaned_data['username'],
+                                    password=form.cleaned_data['password1'],
+                                    )
```
Comment (ruands) [inline authentication_service/views.py]: I may be wrong, but this indentation looks off.

**Comment by ruands** [comment]
+1

File: authentication_service/views.py (hunk 1 above)
Comment (cobusc) [reply authentication_service/views.py]: One off, thanks

Comment (cobusc) [reply authentication_service/views.py]: Oddly enough, PyCharm is suggesting this indentation.

**Pull request merged**